<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="FOCUS-QA-4" name="com.focus.testcases.ReportsTest" tests="23" failures="1" timestamp="22 Jun 2021 16:50:35 GMT" time="546.185" errors="0">
  <testcase name="checkStockStatementReport" time="24.516" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkAllTransactionsOfDocumentClassReport" time="17.300" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkStockAnalysisByRmaReports" time="30.290" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkSavingReportDesignerOfAllTransactionsOfDocumentClass" time="36.638" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkListingOfDocumentsReport" time="16.245" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkReceiptRegisterReport" time="20.172" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkPdcPaymentsRegisterReport" time="19.994" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkProfitAndLossReport" time="29.253" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkStockMovementReports" time="20.099" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkVendorBillWiseSummaryReport" time="19.698" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkTrailBalanceReport" time="47.256" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkStockValuationReports" time="16.056" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkBalanceSheetReport" time="46.765" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkStockDetailsByBinsReports" time="12.589" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkMultiLevelStockMovementReports" time="16.662" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkStockLedgerReport" time="38.843" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkFinanacialLedgerReport" time="22.885" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkCustomerListingOfOutstandingBillsReport" time="12.965" classname="com.focus.testcases.ReportsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.ReportsTest.checkCustomerListingOfOutstandingBillsReport(ReportsTest.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkCustomerListingOfOutstandingBillsReport -->
  <testcase name="checkSubledgerReport" time="19.637" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkVendorAgeingSummaryReport" time="25.749" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkCustomerStatementReport" time="20.421" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkStockAnalysisByBatchReport" time="12.193" classname="com.focus.testcases.ReportsTest"/>
  <testcase name="checkPdcReceiptsRegisterReport" time="19.959" classname="com.focus.testcases.ReportsTest"/>
</testsuite> <!-- com.focus.testcases.ReportsTest -->
